src/urllib3/exceptions.py

```diff
@@ -15,26 +22,33 @@ class HTTPWarning(Warning):
     pass
 
 
+ReduceResult = Tuple[Callable[..., object], Tuple[object, ...]]
```

Comment:
This isn't strictly complete, as technically-speaking, the callable must have the same number of arguments as the tuple has members, but Python's typing doesn't have a way to describe that relationship.